src/helpers/batfish/__tests__/fixtures/data.json

```diff
@@ -29,7 +29,8 @@
           "changelogLink": "/dr-ui/changelog/",
           "installLink": "https://github.com/mapbox/dr-ui/blob/main/README.md",
           "ghLink": "https://github.com/mapbox/dr-ui",
-          "theme": "bg-purple-faint"
+          "lightText": true,
+          "theme": "bg-purple-deep"
```

Comment:
This is generated file when you run `npm run start-docs` that we use for tests. It got behind, but that's ok!